### ISSUE TITLE
doesnt work for china client patch 5.11

### ISSUE THREAD

**Issue by Beatrice-Am** [purutu/ACT.DFAssist#6]
Tried a lot of duties in 5.11 and 5.11hotfix, got no any message.

[14:17:15/Info] Client version: 5.11HF, 00E3/0164/02B0
[14:29:00/Info] Client version: 5.11, 00E3/0164/032D

**Comment by purutu** [comment]
Oops, sorry Beatrice.

I'm not in situation for testing Chinese version of client right now. :(
My IP is blocked and, I don't know why.

I'd like to find Chinese VPN service for testing.

**Comment by purutu** [comment]
I have no way to connect Chinese server now.
Someday I'll try to find a way.

**Comment by MZTheArchMage** [comment]
Could  you please support Chinese server? 
I heard that NeteaseUU VPN and Xunyou VPN can connect CN server.
They are specially prepared for international students studying abroad to play domestic games VPN.
It's been 5.31 now in CN server. looking forward to using it. Thanks!

**Comment by purutu** [comment]
> 
> 
> Could you please support Chinese server?
> I heard that NeteaseUU VPN and Xunyou VPN can connect CN server.
> They are specially prepared for international students studying abroad to play domestic games VPN.
> It's been 5.31 now in CN server. looking forward to using it. Thanks!

Thank you for the information.
Actually I still have problems. I couldn't find account registration and download URL on Chinese service website.
I'll trying to find them again. 
If you don't mind, please let me know the account registration URL. (probably here? https://ff.web.sdo.com)
Thank you

**Comment by purutu** [comment]
I found registration form. 
But I couldn't join

![image](https://user-images.githubusercontent.com/47320226/105533652-c717bd80-5d2f-11eb-997b-65b2f10ac434.png)

This website requested my personal social number(probably). But I don't have it
If you know registration method for foreigner, please let me know.....

**Comment by MZTheArchMage** [comment]
registration address：
https://register.sdo.com/register/index?appId=201&areaId=200&serviceUrl=http%3a%2f%2fwww.sdo.com
![image](https://github.com/MZTheArchMage/pic/blob/main/PIC1.png)
This is an ID searched by Baidu [name:向玉宇  ID:41080119930228457X]  
You can download the game in this website:
https://ff.web.sdo.com/web8/index.html#/home
![image](https://github.com/MZTheArchMage/pic/blob/main/PIC2.png)
Hope this helps you.

**Comment by purutu** [comment]
Thank you for the URL.
It seems like only domestic Chinese could be joining. 
I tried but, couldn't without social identification number. It would be so.
![image](https://user-images.githubusercontent.com/47320226/105606937-57f9a200-5ddf-11eb-89ea-0c32f39f2a94.png)

I have another plan. I'll make a packet detector. Please be patient until job finish

**Comment by purutu** [comment]
You've been waiting a long time!
I just finished support tool now.

https://github.com/purutu/FFXIV.DFASupp (The prebuilt available in Release section)

Would you lease try this tool and post its result here!
Thanks!

![image](https://user-images.githubusercontent.com/47320226/106538356-7b3ef280-653f-11eb-8849-055e08ff839b.png)

**Comment by purutu** [comment]
In Bozja Critical Engagement, There are so many variables that I can't determine methods now. T_T

**Comment by MZTheArchMage** [comment]
###
OpFate = 0x0,
FateIndex = 0,
OpDuty = 0x0,
DutyRoulette = 0,
DutyInstance = 0,
OpMatch = 0x0,
MatchRoulette = 0,
MatchInstance = 0,
OpInstance = 0x0,
OpBozjaCe = 0x0,
###
I test the tool in Mddle La Noscea and The Lochs,  the Result  and Data is empty.
It's seems like can not connect to the game.

**Comment by purutu** [comment]
Oh, probably blocked by firewall.
Please allow the problem to your firewall exception and, test again :)

**Comment by MZTheArchMage** [comment]
> Oh, probably blocked by firewall.
> Please allow the program to your firewall exception and, test again :)
> 
> The program contains Middle La Noscea data only :)

I turn down the firewall, and run as administrator, but it still doesn't work. T T

**Comment by purutu** [comment]
Oh,  understood. Thank you for your cooperation.

Ok, another plan. I will convert the support tool to work in ACT. 
It will work because all data is fetched from the FFXIV plugin.

Please wait again. I'm so sorry.

**Comment by MZTheArchMage** [comment]
> Oh, understood. Thank you for your cooperation.
> 
> Ok, another plan. I will convert the support tool to work in ACT.
> It will work because all data is fetched from the FFXIV plugin.
> 
> Please wait again. I'm so sorry.

Thank you for your work and no need to feel sorry :)
